README.md

```diff
@@ -572,6 +574,54 @@ We have a small collection of algorithms, one for every day of the month. Scroll
       ```
 
 
+## **December 23 - Exam Seating Arrangement**
+  - **Problem**
+    - Exams happen all the time in our education system and a huge amount of effort is put into planning the seating arrangement to prevent students form malpractices. The task here is to create a system that does this automatically when the number of departments, the number of classrooms, and the students roll numbers are given and automatically generate the output plan.
+    - There is only one condition when it comes to seating the students is that **No two students of the same department should sit together**
+
+
+  - **Example**
```

Comment:
So there's a lot of inputs to keep track of, I think we can lose the starting roll number requirement and just let it start from automatically. 

Also add a line stating how the notation works. It's obvious, but better to be safe.